_data/people_list.yml

```
@@ -0,0 +1,618 @@
+  website: https://math.rptu.de/ags/agag/personen/mitglieder
+  email: john.abbott@rptu.de
+  github: JohnAAbbott
+  paid_by_dfg: true
+  status: active
+
+- name: Albin Ahlbäck
+  affiliation: Chalmers University
+  website: https://albinahlback.gitlab.io/
+  email: albin.ahlback@gmail.com
+  github: albinahlback
+  status: active
+
+- name: Alex J. Best
+  affiliation: King's College London
+  website: https://alexjbest.github.io/
+  email: alex.j.best@gmail.com
+  github: alexjbest
+  status: active
+
+- name: Martin Bies
+  affiliation: University of Kaiserslautern-Landau
+  website: https://martinbies.github.io
+  email: martin.bies@rptu.de
+  github: HereAround
+  status: active
+
+- name: Thomas Breuer
+  affiliation: RWTH Aachen University
+  website: http://www.math.rwth-aachen.de/~Thomas.Breuer/
+  email: thomas.breuer@math.rwth-aachen.de
+  github: ThomasBreuer
+  paid_by_dfg: true
+  status: active
+
+- name: Tobias Boege
+  affiliation: KTH Stockholm
+  website: https://taboege.de/
+  email: post@taboege.de
+  github: taboege
+  status: active
+
+- name: Janko Böhm
+  affiliation: University of Kaiserslautern-Landau
+  website: https://agag-jboehm.math.rptu.de/~boehm/
+  email: boehm@mathematik.uni-kl.de
+  github: jankoboehm
+  status: active
+
+- name: Tobias Braun
+  affiliation: RWTH Aachen
+  website: https://www.math.rwth-aachen.de/~Tobias.Braun/
+  email: tobias.braun2@rwth aachen
+  github: SirToby25
+  status: active
+
+- name: Andrei Comaneci
+  github: andreicomaneci
+  website: https://page.math.tu-berlin.de/~comaneci/
+  email: comenaci@math.tu-berlin.de
+  affiliation: TU Berlin
+  status: active
+
+- name: Antony Della Vecchia
+  affiliation: TU Berlin
+  website: https://antonydellavecchia.github.io/
+  email: vecchia@math.tu-berlin.de
+  github: antonydellavecchia
+  status: active
+
+- name: Alexander Demin
+  affiliation: HSE University
+  website: https://sumiya11.github.io/
+  email: asdemin_2@edu.hse.ru
+  github: sumiya11
+  status: active
+
+- name: Berenike Dieterle
+  affiliation: University of Kaiserslautern-Landau
+  github: BD-00
+  email: dieterle@mathematik.uni-kl.de
+  status: active
+  comment: Master student of Claus Fieker as of 2024
+
+- name: Christian Eder
+  affiliation: University of Kaiserslautern-Landau
+  website: https://www.mathematik.uni-kl.de/~ederc/index.html
+  email: ederc@rptu.de
+  github: ederc
+  status: active
+
+- name: Nicolas Faross
+  github: pinguly
+  affiliation: Saarland University
+  email: faross@math.uni-sb.de
+  website: https://www.uni-saarland.de/lehrstuhl/weber-moritz/team/nicolas-faross.html
+  status: active
+  comment: who ?
+
+- name: Kamillo Ferry
+  affiliation: TU Berlin
+  website: https://kafe.dev/
+  email: kafe@kafe.dev
+  github: ooinaruhugh
+  status: active
+
+- name: Ghislain Fourier
+  affiliation: RWTH Aachen University
+  website: https://www.art.rwth-aachen.de/cms/mathb/der-lehrstuhl/team/professorinnen-und-professoren/~rnko/fourier/?allou=1
```

```suggestion
  website: https://www.art.rwth-aachen.de/cms/~rnko/
```
More persistent url (same for other people of our chair below)